assets/openobserve/ingest.py

```
@@ -0,0 +1,53 @@
+
+            if len(data)>50000:
+                res = requests.post(openobserve_url, headers=headers, data=json.dumps(data))
+                data = []
+        if data:
+            print(len(data))
+            res = requests.post(openobserve_url, headers=headers, data=json.dumps(data))
+
+if __name__ == "__main__":
+    try:
+        ingest_dataset()
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)